src/blog/multiPage.html

```diff
@@ -95,7 +95,7 @@
               </div>
             </div> <!-- end col -->
 
-            <div class="menu-socials hidden-sm hidden-xs">
+            <div class="menu-socials hidden-sm hidden-xs" style="display:none;">
```

Comment:
I'm a huge fan of deleting, if you're cool with that.  We can always go back to this commit to find the original html.  Thoughts?